CensusIncomePrediction.ipynb

```diff
@@ -5,9 +5,13 @@
    "metadata": {},
    "source": [
     "# Census Income Prediction with ThirdAI's Universal Deep Transformer\n",
-    "This notebook shows how to build an income prediction model with ThirdAI's Universal Deep Transformer (UDT) model, our all-purpose classifier for tabular datasets. In this demo, we will train and evaluate the model on the CLINC 150 dataset, but you can easily replace this with your own dataset.\n",
+    "This notebook shows how to build an income prediction model with ThirdAI's Universal Deep Transformer (UDT) model, our all-purpose classifier for tabular datasets. In this demo, we will train and evaluate the model on the Census Income dataset, but you can easily replace this with your own dataset.\n",
     "\n",
-    "To run this notebook, you will need to obtain a ThirdAI license at the following link if you have not already: https://www.thirdai.com/try-bolt/"
+    "You can immediately run a version of this notebook in your browser on Google Colab at the following link:\n",
+    "\n",
+    "https://githubtocolab.com/ThirdAILabs/Demos/blob/main/CensusIncomePrediction.ipynb\n",
```

Comment:
Do you prefer the `githubtocolab` link to the final `colab.research.google` version? I think I like the latter more.